tests/cases.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+"""Test cases for PyKEEN."""
+
+import tempfile
+import timeit
+import unittest
+from typing import ClassVar, Optional, Type
+
+from requests.exceptions import ConnectionError
+
+from pykeen.datasets.base import LazyDataSet
+from pykeen.triples import TriplesFactory
+
+
+class DatasetTestCase(unittest.TestCase):
+    """A test case for quickly defining common tests for datasets."""
+
+    #: The expected number of entities
+    exp_num_entities: ClassVar[int]
+    #: The expected number of relations
+    exp_num_relations: ClassVar[int]
```

Comment:
Add `exp_num_triples`? :wink:

Reply:
I could do that

Reply:
Done in e8797936. I need a cross-check that I put all of the right numbers though